### ISSUE TITLE
NVDA Fails to Recognize Links with Embedded  <strong>  Tag Using NVDA+K

### ISSUE THREAD

**Issue by cary-rowen** [nvaccess/nvda#17363]
### Steps to reproduce:
1. Open an HTML file in a browser containing an `<a>` tag with an embedded `<strong>` tag (see code example below).
2. Move focus to the link containing the `<strong>` tag.
3. Press NVDA+K.

```html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>NVDA Test</title>
</head>
<body>
    <p>
        <a href="https://example.com"><strong>Example Link with Strong Tag</strong></a>
    </p>
</body>
</html>
```

### Actual behavior:
NVDA does not recognize this element as a link.

### Expected behavior:
NVDA should correctly identify this element as a link and report href value.

#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2024.4
#### Windows version:
Windows 10 22H2 (AMD64) build 19045.5011
#### Name and version of other software in use when reproducing the issue:
Chrome 130.0.6723.92
#### Other information about your system:
None
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, reproducibility is possible with earlier versions of NVDA that contain this feature.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes
